exporter/sematextexporter/es.go

```diff
@@ -0,0 +1,148 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sematextexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sematextexporter"
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"strings"
+	"time"
+
+	json "github.com/json-iterator/go"
+	"github.com/olivere/elastic/v7"
+	"github.com/sirupsen/logrus"
```

Comment:
"github.com/sirupsen/logrus" is in maintenance mode. Lets use something that is being used by other exporters as well like "go.uber.org/zap". See them for the appropriate usage.

Reply:
Done